content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md

```diff
@@ -21,10 +21,15 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
+3.  Run `kubeadm init` with `--config <YOUR CONFIG YAML>`.
 
 For more details on each field in the configuration you can navigate to our
 [API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2#ClusterConfiguration).
 
+{{< note >}}
+You can get default `ClusterConfiguration` object by `kubeadm config print init-defaults`.
```

Comment:
```suggestion
You can get a starting point ClusterConfiguration by running `kubeadm config print init-defaults` and saving the output to a file of your choice.
```
Any better @Coderhypo ? What do you think?

Reply:
Yes, great. thanks.